syntax = "proto3";
package dbclient;




import "google/protobuf/empty.proto";

/*
DeleteClient请求
*/
message DeleteClientRequest {
    /*
    客户端IP
    */
    string clientIp = 1 ;
}

/*
DeleteClientApi返回
*/
message DeleteClientResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    google.protobuf.Empty data = 4 ;
}

